### PR TITLE
Ratification Polls 04 11

### PR DESCRIPTION
**Pull request by Hipogeo**
@Blimpa

### CODE REVIEW

File: governance/polls/Ratification Poll - Add Data Insights Core Unit Budget (MIP40c3-SP64) - April 11, 2022.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP64 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP64 adds the budget for Data Insights Core Unit (DIN-001) to account for the new mandate 2 that expands on the initial mission to maintain the existing data assets and build new datasets to cover L2s, RWAs, and puts a special focus on the community.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Add Data Insights Core Unit Budget (MIP40c3-SP64) - April 11, 2022.md:30]: Fix the format here

File: governance/polls/Ratification Poll - Add Data Insights Core Unit Budget (MIP40c3-SP64) - April 11, 2022.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+59,500 DAI → 90,250.00 DAI per month (+52%)
+6 month budget → 12 month budget
+Team headcount increase
+Expansion Pool
+MKR vesting program
+Travel budget*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/25c4b48d8e0f6d2ccee517df0555f0766feff8a5/MIP40/MIP40c3-Subproposals/MIP40c3-SP64.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Add Data Insights Core Unit Budget (MIP40c3-SP64) - April 11, 2022.md:42]: There's a later commit now. Please modify :)

File: governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: Ratification Poll for Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022
+summary: Regarding the offboard of the Content Production CU, this poll is for removing their wallet from the recognized multi-sig list.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md:3]: I propose an alternative wording: "This subproposals removes Content Production's (MKT-001) multi-sig from the MIP47 list of recognized MakerDAO multi-sigs"

File: governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP47c3-SP1 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Now that the Content Production Core Unit has been offboarded, their wallet should be removed from the recognized multi-sig list. Currently, no funds are available in the multi-sig, but it can still claim ~3.8k DAI from their canceled budget stream.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md:30]: Fix format

File: governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+Seeing as the Content Production Core Unit is no longer an active Core Unit for the DAO, there is only downside to leaving their multi-sig Governance recognized. Given the small amount of funds that can still be claimed, there is little danger to publicly announce our intentions to close the loop with the DssVest stream. However, this process is clearly vulnerable to front running until the procedure for reclaiming funds and permissions is improved.
+
+If this subproposal is accepted, there will be the technical work of claiming the budget funds and returning them to the surplus buffer, to be completed in a future executive proposal.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/25c4b48d8e0f6d2ccee517df0555f0766feff8a5/MIP47/MIP47c3-Subproposals/MIP47c3-SP1.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md:37]: There's a later commit (with a very minor correction) now

File: governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+* [Latest Proposal Version](Can't Find One)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md:38]: That's what I corrected :)

Here it is: https://mips.makerdao.com/mips/details/MIP47c3SP1

File: governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
+---
+title: Ratification Poll for Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022
+summary: This poll is for accepting the updates to the mandate for Core Unit COM-001.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md:3]: "This subproposal refreshes the mandate for the Governance Communications Core Unit (COM-001)."

File: governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP39c2-SP30 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Governance Communications Core Unit performs a set of programs that focus on MakerDAO’s informational accessibility, stakeholder coordination, and communication practices as a decentralized and public organization.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md:30]: I would use this instead:

"MIP39c2-SP30 updates the mandate for Core Unit COM-001. The Governance Communications Core Unit performs a set of programs that focus on MakerDAO's informational accessibility, stakeholder coordination, and communication practices as a decentralized and public organization."

Comment (blimpa) [reply governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md:30]: Also, fix format

File: governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+Has well-managed public communication venues.
+Engages with stakeholders to improve its coordination & output.
+Advocates for, facilitates, and promotes beneficial organizational communication practices.
+Maps the organization, its initiatives, and the surrounding stakeholder ecosystem.
+Maintains resources and processes for crisis situations.
+Provides dedicated support and multiple points of access to members of the media.
+Results in a more effective and less opaque organization that attracts more stakeholders.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/25c4b48d8e0f6d2ccee517df0555f0766feff8a5/MIP39/MIP39c2-Subproposals/MIP39c2-SP30.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md:46]: Later commit: https://github.com/makerdao/mips/blob/0fa58d68159bb06950d15154682a73d2b6c46fe6/MIP39/MIP39c2-Subproposals/MIP39c2-SP30.md

File: governance/polls/Ratification Poll - Modify Protocol Engineering Core Unit Budget, PE-001 (MIP40c3-SP68) - April 11, 2022.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
+---
+title: Ratification Poll for Modify Protocol Engineering Core Unit Budget, PE-001 (MIP40c3-SP68) - April 11, 2022
+summary: This subproposal is requesting a 12 month Budget of 7,590,000 DAI for the Protocol Engineering Core Unit starting from May 2022.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Modify Protocol Engineering Core Unit Budget, PE-001 (MIP40c3-SP68) - April 11, 2022.md:3]: If we're gonna include the start date here, we should also include the end date, I think

File: governance/polls/Ratification Poll - Modify Protocol Engineering Core Unit Budget, PE-001 (MIP40c3-SP68) - April 11, 2022.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP68 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Protocol Engineering would like to secure an operating budget for it to deliver its mandate over the next 12 months.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Modify Protocol Engineering Core Unit Budget, PE-001 (MIP40c3-SP68) - April 11, 2022.md:30]: Fix format

File: governance/polls/Ratification Poll - Update Data Insights Core Unit Mandate (MIP39c2-SP31) - April 11, 2022.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP39c2-SP31 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Data is important. The more data we have at our disposal, the better position we’ll be in to make good decisions and take advantage of new opportunities. Good data will also give us the justification and evidence we need to back up these decisions.
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Update Data Insights Core Unit Mandate (MIP39c2-SP31) - April 11, 2022.md:30]: "This subproposal modifies the Data Insights Core Unit (“DIN”) mandate previously ratified with [MIP39c2-SP22](https://forum.makerdao.com/t/mip39c2-sp22-adding-data-insights-core-unit-din-001/10208). It’s mostly about simplification of the previous document and stressing of our focus on L2 and cross-chain data as well as RWA datasets"

Also, fix format

**Review by blimpa — APPROVED** [review]
LGTM

@prose11 , @LongForWisdom

**Review by prose11 — COMMENTED** [review]
Few changes to make, will be submitted in my PR for all the polls

**Pull request merged**